features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -889,6 +889,116 @@
       {
           "id": "QWNjb3VudERpc2FibGU",
           "displayName": "Account Disabled"
+      },
+      {
+          "id": "RW1haWxDb25maXJt",
+          "displayName": "Email Confirmation",
+          "description": "This email is used to request the user to confirm their email address"
+      },
+      {
+          "id": "QWNjb3VudElkUmVjb3Zlcnk",
+          "displayName": "Recover user account",
+          "description": "This email is sent when a user accounts needs to be recovered"
+      },
+      {
+          "id": "VGVtcG9yYXJ5UGFzc3dvcmQ",
+          "displayName": "Temporary Password",
+          "description": "This email is used to sent a temporary password to the user"
+      },
+      {
+          "id": "T25lVGltZVBhc3N3b3Jk",
+          "displayName": "One-Time Password",
+          "description": "This email is used to sent a One-Time Password (OTP) to the user"
+      },
+      {
+          "id": "cmVzZW5kQXNrUGFzc3dvcmQ",
+          "displayName": "Resend Ask Password",
+          "description": "This email is send when the organization admin reinitiates the Ask Password for a user"
+      },
+      {
+          "id": "cmVzZW5kQWRtaW5Gb3JjZWRQYXNzd29yZFJlc2V0",
+          "displayName": "Resend Admin-Requested Forced Password Reset",
+          "description": "This email is sent when the organization admin reinitiates the forced password reset for a user"
+      },
+      {
+          "id": "QWRtaW5Gb3JjZWRQYXNzd29yZFJlc2V0V2l0aE9UUA",
+          "displayName": "Admin Forced Password Reset with OTP",
+          "description": "This email is sent when the organization admin initiates the forced password reset for a user with OTP"
+      },
+      {
+          "id": "cmVzZW5kQWRtaW5Gb3JjZWRQYXNzd29yZFJlc2V0V2l0aE9UUA",
+          "displayName": "Resend Admin Forced Password Reset with OTP",
+          "description": "This email is sent when the organization admin reinitiates the forced password reset for a user with OTP"
+      },
+      {
+          "id": "YWNjb3VudEFjdGl2YXRpb25TdWNjZXNz",
+          "displayName": "Account Activation Confirmation",
+          "description": "This email notifies the user that the account activation is successful"
+      },
+      {
+          "id": "aW5pdGlhdGVSZWNvdmVyeQ",
+          "displayName": "Password Reset Initiation",
+          "description": "This email is sent when the user initiates a password reset"
+      },
+      {
+          "id": "aWRsZUFjY291bnRSZW1pbmRlcg",
+          "displayName": "Idle Account Reminder",
+          "description": "This email is sent when the user account is idle for a long time"
```

Comment:
instead of `a long time`, shall we say something like `a configured amount of time`?

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/commit/06fb9e85aaa8d95bd511be187d382a392382cc99